<ui version="4.0" >
 <class>NMessageDialog</class>
 <widget class="QDialog" name="NMessageDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>70</number>
   </property>
   <property name="rightMargin" >
    <number>70</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="titleLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>230</red>
           <green>230</green>
           <blue>230</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>30</number>
     </property>
     <property name="leftMargin" >
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="iconLabel" >
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="textScrollArea" >
       <property name="font" >
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="verticalScrollBarPolicy" >
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy" >
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable" >
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>379</width>
          <height>138</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="textLabel" >
           <property name="font" >
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="wordWrap" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
